lib/verify.js

```diff
@@ -66,28 +100,38 @@ export function isExpired({credential, now = new Date(), maxClockSkew}) {
 }
 
 export async function verify({
-  credential, capability, variables = {}, options
+  credential, presentation, capability, variables = {}, options
 } = {}) {
   // set default options
   options = {
     checkExpiration: !options?.returnExchange,
     maxClockSkew: 300,
     returnExchange: false,
     getCredentialFromExchange: _getCredentialFromExchange,
+    getPresentationFromExchange: _getPresentationFromExchange,
     ...options
   };
   const {
-    checkExpiration, maxClockSkew, returnExchange, getCredentialFromExchange
-  } = options;
+    checkExpiration, maxClockSkew, returnExchange, getCredentialFromExchange,
+    getPresentationFromExchange} = options;
   assert.bool(checkExpiration, 'options.checkExpiration');
   assert.number(maxClockSkew, 'options.maxClockSkew');
   assert.bool(returnExchange, 'options.returnExchange');
   assert.func(getCredentialFromExchange, 'options.getCredentialFromExchange');
+  assert.func(getPresentationFromExchange,
+    'options.getPresentationFromExchange');
```

Comment:
Style is: If any function param gets dropped, drop them all and group them together. Single line or multiple line grouping thereafter is fine, but the params get grouped together, don't group the first param with the name of the function and then do the others differently as it can harm readability.

```suggestion
  assert.func(
    getPresentationFromExchange, 'options.getPresentationFromExchange');
```

Reply:
@dlongley I would value a longer conversation about styling. Do we have a standard that is written down that we wrote and/or reference? As you can tell, the our styling is not intuitive to me (which might just be a 'me' problem) and often leaves me guessing as to what other's will expect to see. On the other hand, there are other standards and formatters out there that I do find more intuitive and consistent.